src/OpenLoco/Graphics/Gfx.h

```diff
@@ -277,7 +277,10 @@ namespace OpenLoco::Gfx
 
     G1Element* getG1Element(uint32_t id);
 
-    void setCurrentFontSpriteBase(int16_t value);
```

Comment:
setter already existed here, and looking at where it's used, it makes sense to make this the new home